README.md

```diff
@@ -1,22 +1,34 @@
-# Cellar Rebalancer
+# Steward
 
-The Cellar Rebalancer is an appliaction intended for developers and validators on Sommelier network.
+Steward is an application intended for developers and validators on Sommelier network.
 
 It can run in single signer mode or as a voter in the Cosmos sommelier protocol.
 
-It integrates the full functionality of gorc for operating as an orchestator and relayer of gravity bridge messages between Ethereum and Cosmos chains.
+It integrates the full functionality of gorc for operating as an orchestrator and relayer of gravity bridge messages between the Ethereum and Cosmos chains.
+
+Steward is built with the [Abscissa](https://github.com/iqlusioninc/abscissa) app micro-framework.
 
 ## TLS
 
-To provide data to Steward, an encrypted and authenticated gRPC connection must be established. The client certificate authority used by the initial Data Provider is included in `tls/`. This is the only client root of trust accepted by default by Steward right now as we are only accepting client certs from one Data Provider, VolumeFi.
+To provide data to Steward, an encrypted and authenticated gRPC connection must be established. The client certificate authority used by the initial Data Provider is included in `tls/`. This is the only client root of trust accepted by default in Steward right now as we are only accepting client certs from one Data Provider, [VolumeFi](https://volume.finance/).
+
+## Steward use case
```

Comment:
I think there are three Steward use cases:

1. Accepting recommendations for executing strategies from data providers and then voting on the Sommelier chain
2. Running the orchestrator/relayer
3. Providing a suite of utility functions for interacting with Sommelier and the gravity bridge

We cover number 2 here, and mostly cover number 1, except only implicitly. It would be good to mention explicitly that Steward is providing a gRPC server to accept these recommendations and then go into the detail of how those get set up. We don't mention the existence of the third use case until the list of subcommands further down.

Reply:
fixed